wolfcrypt/src/asn.c

```diff
@@ -6309,6 +6404,26 @@ static int ConfirmSignature(SignatureCtx* sigCtx,
             #ifndef NO_RSA
                 case RSAk:
                 {
+                #ifdef WOLFSSL_RENESAS_TSIP_TLS
```

Comment:
Should the TSIP section here go below the HAVE_PK_CALLBACKS section?  Otherwise TSIP users who want to use Public Key Callbacks (PK Callbacks) won't be able to.

Reply:
OK. Your suggestion should be true. I thought when user enabled RENESAS_TSIP_TLS definition, user would want to primally use the TSIP capability first even though enabled PK Callbacks .

Moved the section just after PK Callbacks